integration-tests/src/test/java/org/apache/druid/tests/query/ITWikipediaQueryTest.java

```diff
@@ -51,15 +69,78 @@ public void before() throws Exception
     ITRetryUtil.retryUntilTrue(
         () -> coordinatorClient.areSegmentsLoaded(WIKIPEDIA_DATA_SOURCE), "wikipedia segment load"
     );
-    coordinatorClient.initializeLookups(WIKIPEDIA_LOOKUP_RESOURCE);
-    ITRetryUtil.retryUntilTrue(
-        () -> coordinatorClient.areLookupsLoaded(WIKI_LOOKUP), "wikipedia lookup load"
-    );
+    if (!coordinatorClient.areLookupsLoaded(WIKI_LOOKUP)) {
+      coordinatorClient.initializeLookups(WIKIPEDIA_LOOKUP_RESOURCE);
+      ITRetryUtil.retryUntilTrue(
+          () -> coordinatorClient.areLookupsLoaded(WIKI_LOOKUP), "wikipedia lookup load"
+      );
+    }
   }
 
   @Test
   public void testWikipediaQueriesFromFile() throws Exception
   {
     queryHelper.testQueriesFromFile(WIKIPEDIA_QUERIES_RESOURCE, 2);
   }
+
+  @Test
```

Comment:
Should we only have a test for when the Manual Lane From Context That Arent In Map is given? For example, if lane in query context is "some-unknown-lane"

Reply:
I guess they will always be rejected since we give NO_CAPACITY right?

Reply:
^ update typo above. Meant to say "also" have a test for case when the Manual Lane From Context That Arent In Map is given